pages/table/sticky-columns.page.tsx

```diff
@@ -0,0 +1,318 @@
+    alt: 'Sixth',
+    description: 'This is the sixth item',
+    type: '1A',
+    size: 'Small',
+  },
+];
+
+export default () => {
+  const [preferences, setPreferences] = React.useState<CollectionPreferencesProps.Preferences>({
+    stickyColumns: { first: 1, last: 1 },
+  });
+
+  return (
+    <ScreenshotArea>
+      <h1>Sticky columns</h1>
+      <SpaceBetween size="xl">
+        <Table
+          data-test-id="simple"
+          stickyColumns={{ first: 1, last: 1 }}
+          columnDefinitions={COLUMN_DEFINITIONS}
+          items={ITEMS}
+          sortingDisabled={true}
+          ariaLabels={{
+            tableLabel: 'simple',
+          }}
+          header={<Header>Simple table</Header>}
+        />
+        <Table
+          data-test-id="resizable"
+          resizableColumns={true}
+          stickyColumns={{ first: 1, last: 1 }}
+          columnDefinitions={COLUMN_DEFINITIONS}
+          items={ITEMS}
+          sortingDisabled={true}
+          ariaLabels={{
+            tableLabel: 'Resizable',
+          }}
+          header={<Header>Resizable columns</Header>}
+        />
+        <Table
+          data-test-id="with-collection-preferences"
+          stickyColumns={preferences.stickyColumns}
+          stickyHeader={true}
+          preferences={
+            <CollectionPreferences
+              title="Preferences"
+              confirmLabel="Confirm"
+              cancelLabel="Cancel"
+              onConfirm={({ detail }) => setPreferences(detail)}
+              preferences={preferences}
+              stickyColumnsPreference={{
+                firstColumns: {
+                  title: 'Stick first column(s)',
+                  description: 'Keep the first column(s) visible while horizontally scrolling table content.',
+                  options: [
+                    { label: 'None', value: 0 },
+                    { label: 'First column', value: 1 },
+                    { label: 'First two columns', value: 2 },
+                  ],
+                },
+                lastColumns: {
+                  title: 'Stick last column',
+                  description: 'Keep the last column visible while horizontally scrolling table content.',
+                  options: [
+                    { label: 'None', value: 0 },
+                    { label: 'Last column', value: 1 },
+                  ],
+                },
+              }}
+            />
+          }
+          ariaLabels={{
+            selectionGroupLabel: 'Items selection',
+            allItemsSelectionLabel: ({ selectedItems }) =>
+              `${selectedItems.length} ${selectedItems.length === 1 ? 'item' : 'items'} selected`,
+            itemSelectionLabel: ({ selectedItems }, item) => {
+              const isItemSelected = selectedItems.filter(i => i.name === item.name).length;
+              return `${item.name} is ${isItemSelected ? '' : 'not'} selected`;
+            },
+            tableLabel: 'With collection preferences',
```

Comment:
Would it make more sense to add collection preferences in the page's top that would change preferences for all tables? That would make it easier to test. In that case can also remove that extra table that only features having a preferences included.